controllers/watcher_controller.go

```diff
@@ -240,6 +257,10 @@ func (r *WatcherReconciler) initConditions(instance *watcherv1beta1.Watcher) err
 			watcherv1beta1.WatcherRabbitMQTransportURLReadyCondition,
 			condition.InitReason,
 			condition.RabbitMqTransportURLReadyInitMessage),
+		condition.UnknownCondition(
```

Comment:
In line 207  we're marking `InputReadyCondition` True, but that condition is not initalized here